### PR TITLE
Fixes layout issues with certain UITableViews on iOS 9

### LINKED ISSUE #4277: Layout issues on iPad running iOS 9
It looks there is some layout problems on iOS 9, maybe related to the hacks on WPTableViewCell.

simulator screen shot 15 sep 2015 22 41 32

### PR DESCRIPTION
**Pull request by aerych**
Closes #4277 

By default, iOS 9 configures `UITableView.cellLayoutMarginsFollowReadableWidth` to true, causing issues with the layout hack in `WPTableViewCell`.  We need to properly address the hack (see #2491) but in the meantime, this PR will fix the layout on iOS 9. 

A few notes:
- I went with a new category in the main project vs editing `WPStyleGuide` in the shared pod as a matter of convenience.  I'm fine moving it to the pod if we feel that's a better fit.
- I only added the margin correction where it seemed to be necessary.  `UITableView`s sporting customized cell layouts don't seem to care either way.
- Rather than add more header imports I added it once to the prefetch.  Seems reasonable in this case due to the number of instances affected. 

cc: @koke @oguzkocer @jleandroperez @astralbodies @diegoreymendez and @bummytime 
I tried to catch all the places where the margins were affecting table view layout but please double check the features you've wrangled in case I missed something I didn't know to look for :x. 

Needs Review: @SergioEstevao

### CODE REVIEW

**Comment by koke** [comment]
The weirdness in the topic selector is gone :)

File: WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift [hunk 1]
```diff
@@ -79,8 +79,12 @@ public class NotificationSettingDetailsViewController : UITableViewController
         
         // Style!
         WPStyleGuide.configureColorsForView(view, andTableView: tableView)
+
+        if #available(iOS 9.0, *) {
```
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift]: Niiiiice!

**Comment by SergioEstevao** [comment]
@aerych Travis is complaining that this test is failing:
```iOS/WordPress/WordPressTest/ReaderStreamViewControllerTests.swift:18:74: missing argument for parameter 'isFeed' in call
        let controller = ReaderStreamViewController.controllerWithSiteID(NSNumber(int: 1))
           ^
```
Don't know if related to these changes specifically but do you mind to have look?

**Comment by aerych** [comment]
Hi @SergioEstevao.  It looks like I broke two tests, one with #4280 and one with #4311. Sorry about that. :(  I'll add the fixes to this PR.

**Comment by aerych** [comment]
@SergioEstevao Ready for another round.

**Comment by oguzkocer** [comment]
The changes look good to me. I've also did a quick test of every page I could think of and didn't see any layout issues. Thanks for taking care of this one @aerych!

**Comment by SergioEstevao** [comment]
:shipit:

**Comment by aerych** [comment]
Thanks @SergioEstevao!

**Comment by aerych** [comment]
@koke, @oguzkocer thanks for the extra set of eyes :)

**Comment by astralbodies** [comment]
Dropped the ball on reviewing this, sorry @aerych.

**Pull request merged**